app/src/swap/handle_check_address.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _HANDLE_CHECK_ADDRESS_H_
+#define _HANDLE_CHECK_ADDRESS_H_
```

Comment:
use `#pragma once`, it's less error prone when copying files around
see `app-exchange/src/buffer.h` for example

Reply:
Ok I'll fix it.

Reply:
Done.